parsl/tests/site_tests/conda_setup.sh

```diff
@@ -75,15 +75,6 @@ create_conda() {
         echo "module load ibm-wml-ce"                    >  ~/setup_parsl_test_env.sh
         echo "conda activate $PWD/$CONDA_TARGET"         >> ~/setup_parsl_test_env.sh
 
-    elif [[ "$(hostname)" =~ .*cori.* ]]
-    then
-        echo "On Cori"
```

Comment:
As an aside, sucks to be bridges: have executed `hostname` n times at that point.